modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -6344,6 +6359,25 @@ else if (interceptorVal != updated0) {
                 transformed);
         }
 
+        /**
+         * Checks, that change was made on another cluster.
+         *
+         * @param conflictCtx Conflict context associated with change.
+         * @return {@code True} If change was made on another cluster (i.e. change was got by DR) and {@code False}
+         * otherwise.
+         */
+        private boolean changesFromAnotherDataCenter(@Nullable GridCacheVersionConflictContext<?, ?> conflictCtx){
```

Comment:
Looks like it's enough to check for `conflictVer != null` to determine if update came from DrReceiver.
Newly introduced method is not needed.
Do it before calling `org.apache.ignite.internal.processors.cache.GridCacheMapEntry.AtomicCacheUpdateClosure#resolveConflict` or the property could be cleared by conflict resolution code.